operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -405,8 +405,11 @@ message GlobalConfig {
 
   // Controls whether one external istiod is enabled.
   google.protobuf.BoolValue externalIstiod = 62;
+  
+  // Controls whether a remote cluster is the config cluster for an external istiod
+  google.protobuf.BoolValue configCluster = 64;
```

Comment:
is this new or we used to have it and now bringing it back?

Reply:
This is a new flag. Previously we used `base.enabled` to pull in the extra charts needed on a config cluster vs. a plain remote cluster, but now the extra charts are part of the istodRemote component and this flag is used to turn them on/off.

Reply:
Makes sense, great. Seems like this will also require changes to our documentation around installing the config cluster?

Reply:
Yep. https://github.com/istio/istio.io/pull/9983